dotnet/Microsoft.iOS.Templates/ios/Info.plist

```diff
@@ -0,0 +1,48 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>CFBundleDisplayName</key>
+    <string>iOSApp1</string>
+    <key>CFBundleIdentifier</key>
+    <string>com.companyname.iOSApp1</string>
+    <key>CFBundleShortVersionString</key>
+    <string>1.0</string>
+    <key>CFBundleVersion</key>
+    <string>1.0</string>
+    <key>LSRequiresIPhoneOS</key>
+    <true/>
+    <key>MinimumOSVersion</key>
+        <string>minOSVersion</string>
+    <key>UIDeviceFamily</key>
+    <array>
+        <!--#if((deviceFamily == "iphone") || (deviceFamily == "universal"))
+        <integer>1</integer>
+        #endif-->
+        <!--#if((deviceFamily == "ipad") || (deviceFamily == "universal"))
+        <integer>2</integer>
+        #endif-->
```

Comment:
^ that expand correctly with the (new) template engine used by dotnet ?

Reply:
Yes, both of these examples work:
```
% dotnet new ios --deviceFamily iPad
% dotnet new ios --deviceFamily ipad
```